library/fixed_point/src/Measurement.qs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import Types.FixedPoint;
+import Convert.BoolArrayAsFixedPoint;
+import Std.Arrays.ForEach;
+
+/// # Summary
+/// Measure a fixed-point number, returns its value as Double, and resets
+/// all the register to zero.
+///
+/// # Input
+/// ## fp
+/// Fixed-point number to measure.
+operation MeasureFxP(fp : FixedPoint) : Double {
+    let bits = ForEach(q => MResetZ(q) == One, fp::Register);
```

Comment:
```suggestion
    let measurements = MResetEachZ(fs::Register);
    let bits = ResultArrayAsBoolArray(measurements);
```

Reply:
This is a suggestion to make the library more friendly to QIR generation. We currently don't support dynamic `Double` types so realistically it won't have any effect immediately, but just something to keep in mind.

Reply:
We can always do an effor to make it more hardware friendly later, so feel free to ignore this comment.